docs/container.md

```diff
@@ -63,12 +64,12 @@ class Config extends ContainerConfig
     {
         $adr = $di->get('radar/adr:adr');
 
+        $adr->middle('Relay\Middleware\ResponseSender');
```

Comment:
ok, no problem. I will try out. Just to make it clear what I asked. From the earlier diff there was this comment 

> Yes, I think ResponseSender. Believe it or not, ResponseSender might be best as the first entry, rather than the last, so it can send any responses that get generated by other middlewares.

So as this line https://github.com/radarphp/Radar.Project/pull/19/files#diff-762e172d280769ca6701a87863ec30f2R87 

```
 $boot = new Boot();
 $adr = $boot->adr([
     'Foo\Bar\ContainerConfig',
     'My\Config',
 ]);
```

the first one is `Foo\Bar\ContainerConfig` .  

```php
namespace Foo\Bar;
 
 use Aura\Di\Container;
 use Aura\Di\ContainerConfig;
 
 class Config extends ContainerConfig
 {
     public function modify(Container $di)
     {
         $adr = $di->get('radar/adr:adr');
         $adr->middle('Foo\Bar\SomeMiddlewareHandler');
     }
}
```

So I was wondering if there is a call to add `Foo\Bar\SomeMiddlewareHandler` then `Relay\Middleware\ResponseSender` may not be the first middleware as mentioned.

Mostly I would have been wrongly interpreted. So I may want to test this and see.

Thank you.